independent-publisher-connectors/Safaricom Mpesa Daraja/README.md

```diff
@@ -0,0 +1,49 @@
+# Daraja api connector
```

Comment:
Hi @johnmuchiriM, 
Please update readme.md file as per the template in this path. 
https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/readme.md